crates/wac-parser/wac.pest

```diff
@@ -85,6 +84,14 @@ InterfaceRef         =  { PackagePath | Ident }
 ExternType           = !{ FuncType | InlineInterface | Ident }
 InlineInterface      =  { InterfaceKeyword ~ InterfaceBody }
 
+WorldIncludeStatement  = ${ IncludeKeyword ~ DelimitingSpace+ ~ WorldIncludeItem ~ DelimitingSpace* ~ Semicolon }
+WorldIncludeItem       = !{ WorldRef ~ DelimitingSpace* ~ WorldIncludeWithClause? }
```

Comment:
As this is non-atmic, we don't need the delimiting space:

```suggestion
WorldIncludeItem       = !{ WorldRef ~ WorldIncludeWithClause? }
```

If we wanted to collapse this into `WorldIncludeStatement`, it would look like

```
WorldIncludeStatement  = ${ IncludeKeyword ~ DelimitingSpace+ ~ WorldRef ~ DelimitingSpace* ~ WorldIncludeWithClause? ~ DelimitingSpace* ~ Semicolon }
```

Reply:
To make it clearer given all the individual suggestions, I think the rules could look like:

```pest
WorldIncludeStatement  = ${ IncludeKeyword ~ DelimitingSpace+ ~ WorldRef ~ DelimitingSpace* ~ WorldIncludeWithClause? ~ DelimitingSpace* ~ Semicolon }
WorldIncludeWithClause = !{ WithKeyword ~ OpenBrace ~ WorldIncludeItem ~ ("," ~ WorldIncludeItem)* ~ ","? ~ CloseBrace }
WorldIncludeItem       = ${ Ident ~ DelimitingSpace+ ~ AsKeyword ~ DelimitingSpace+ ~ Ident }
WorldRef               = { PackagePath | Ident }
```